crates/http-api-bindings/src/completion/mod.rs

```diff
@@ -6,11 +6,17 @@ use llama::LlamaCppEngine;
 use tabby_common::config::HttpModelConfig;
 use tabby_inference::CompletionStream;
 
-pub fn create(model: &HttpModelConfig) -> Arc<dyn CompletionStream> {
-    if model.kind == "llama.cpp/completion" {
-        let engine = LlamaCppEngine::create(&model.api_endpoint, model.api_key.clone());
-        Arc::new(engine)
-    } else {
-        panic!("Unsupported model kind: {}", model.kind);
+pub async fn create(model: &HttpModelConfig) -> Arc<dyn CompletionStream> {
+    match model.kind.as_str() {
+        "llama.cpp/completion" => {
+            let engine = LlamaCppEngine::create(&model.api_endpoint, model.api_key.clone());
+            Arc::new(engine)
+        }
+        "ollama" => ollama_api_bindings::create_completion(model).await,
```

Comment:
```suggestion
        "ollama/completion" => ollama_api_bindings::create_completion(model).await,
```

Reply:
Can you me give a explanation why this suffix is required, please? Below is sample of config part, as you can see, word `completion` appears as an attribute (`model.completion`) as well as in kind name. 

```toml
[model.completion.http]
kind = "ollama/completion"
# ...
```

Reply:
I might have understood the reasoning behind, however, I have a question. Does `/completion` refers to some internal Tabby naming so it will consistent for all bindings or it refers to path in Ollama API? In other words, should I use `/generate` instead because Ollama uses `/api/generate` in fact. See [Ollama API docs](https://github.com/ollama/ollama/blob/main/docs/api.md#generate-a-completion)

Reply:
Mostly because we wanna split the concept of `completion`,  `chat` and `embedding`, the current implementation looks good